libphobos/m4/druntime/libraries.m4

```diff
@@ -57,10 +57,91 @@ AC_DEFUN([DRUNTIME_LIBRARIES_ZLIB],
 
 # DRUNTIME_LIBRARIES_ATOMIC
 # -------------------------
-# Allow specifying whether to use libatomic for
-# 128 bit atomic support.
 AC_DEFUN([DRUNTIME_LIBRARIES_ATOMIC],
 [
+  # Check for atomic builtins up to 64 bit.
```

Comment:
@jpf91 - I've put this in libraries.m4 next to the check for libatomic, would it be better to put this elsewhere? Such as `druntime/cpu.m4`, as it's not strictly library related.